mkdocs.yml

```diff
@@ -37,6 +37,7 @@ nav:
       - ios-predicate.md
       - parallel-tests.md
       - execute-methods.md
+      - clipboard.md
```

Comment:
not sure if this should be added to the troubleshooting list instead

Reply:
or... maybe we may need a section to explain each command..? (not mobile execute)